qt/widgets/mplcpp/test/ZoomerTest.h

```diff
@@ -14,7 +14,6 @@
 
 using MantidQt::Widgets::MplCpp::FigureCanvasQt;
 using MantidQt::Widgets::MplCpp::Zoomer;
-namespace Python = MantidQt::Widgets::MplCpp::Python;
```

Comment:
How about just change this to `namespace Python = MantidQt::Widgets::Common::Python;` to avoid the repeated lengthy namespace calls below?